packages/rainbowkit/src/components/RainbowKitProvider/RainbowKitChainContext.tsx

```diff
@@ -9,10 +8,6 @@ export interface RainbowKitChain {
   iconBackground?: string;
 }
 
-// This type is a combination of wagmi and RainbowKit chain types to make
-// it easier for consumers to define their chain config in a single place.
-export type Chain = WagmiChain & RainbowKitChain;
-
```

Comment:
No longer used. Most of the files which used this type was from [rainbowkit connectors](https://github.com/search?q=repo%3Arainbow-me%2Frainbowkit+%2FRainbowKitChainContext&type=code&p=3)